docs/application/dotnet/guides/nui/nui-components/Scrollbar.md

```diff
@@ -37,6 +38,92 @@ Following output is generated when the progress is created using property:
 
 ![CreateWithProperties](./media/scrollbar_properties.PNG)
 
+## Create with style
+
+To create a scroll bar using style, follow these steps:
+
+1. Create a style for scroll bar and set the properties for track and thumb:
+
+    ```cs
+    ScrollBarStyle style = new ScrollBarStyle
+    {
+        Track = new ImageViewStyle
+        {
+            BackgroundColor = Color.Green
+        },
+        Thumb = new ImageViewStyle
+        {
+            BackgroundColor = Color.Black,
+            Size = new Size(30, 4)
+        },
+    };
+    ```
+
+2. Use the style to create a scroll bar and add it to parent:
+
+    ```cs
+    scrollBar = new ScrollBar(style);
+    scrollBar.Size = new Size(300, 4);
+    scrollBar.MaxValue = (int)scrollBar.SizeWidth / 10;
+    scrollBar.MinValue = 0;
+    scrollBar.CurrentValue = 0;
+    root.Add(scrollBar);
+    ```
+
+Following output is generated when the progress is created using style:
+
+![CreateWithProperties](./media/scrollbar_properties.PNG)
+
+## Create with defined styles
+
+You can define a style based on the user experience (UX) and then use this style to ceate a scroll bar.

```

Comment:
```suggestion
You can define a style based on the user experience (UX) and then use this style to create a scroll bar.
```

Reply:
Global Comment:
@mijin85cho 
Thank you for pointing this.
I think this was missed from the previous comment, https://github.com/Samsung/tizen-docs/pull/1060#discussion_r399030608.